src/components/shared/DomainDropdown/DomainDropdown.tsx

```diff
@@ -90,7 +89,16 @@ const DomainDropdown = ({
     const allDomainsOption: SelectOption = {
       children: (
         <DomainDropdownItem
-          domain={ALLDOMAINS_DOMAIN_SELECTION}
+          domain={{
+            id: '',
+            isRoot: true,
```

Comment:
This is a bit misleading. 

"All teams" include all domains, including root. See it more of a grouping of domains, rather than a nested tree/list.

So maybe leave a comment explaining this so future devs won't be confused

Reply:
This is actually a dummy domain data that we use to display the "All Teams" option in the dropdown. `isRoot` has to be set to true so that it's not shown as a subdomain item like so:
<img width="342" alt="image" src="https://user-images.githubusercontent.com/112586815/221062102-17d48183-e63d-4aab-8c31-143378e451e5.png">

But I agree it's confusing without further explanation so I added a relevant comment